src/command/Commands.ts

```diff
@@ -99,7 +99,19 @@ export abstract class Command {
   abstract handle(params: Parameter<ParameterType>[]): HandleResult;
 
   static getAvailableCommands(): Command[] {
-    return [new UndoCommand(), new RedoCommand(), new AddCommand(), new ConfigCommand(), new HelpCommand()];
+    return [
```

Comment:
Reorder the list of commands from A to Z

Reply:
Updated.